pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
It's not just early return - return with error from this function means that we will not generate anything for that type.
So that PR is effectively no-op for now.